src/components/ChatPanel.tsx

```diff
@@ -107,15 +115,22 @@ export function ChatPanel(props: ChatPanelProps) {
       <div className={cssClasses.container}>
         {header}
         <div className={cssClasses.messagesScrollContainer}>
-          <div ref={messagesRef} className={cssClasses.messagesContainer}>
-            {messages.map((message, index) => (
-              <MessageBubble
-                {...props}
-                customCssClasses={cssClasses.messageBubbleCssClasses}
-                key={index}
-                message={message}
-              />
-            ))}
+          <div ref={messagesContainer} className={cssClasses.messagesContainer}>
+            {messages.map((message, index) => {
+              return (
```

Comment:
nit: you can do `messages.map((message, index) => ({<div>...</div>})` to avoid the explicit `return` keyword.

Reply:
done